grumphp.yml.dist

```diff
@@ -0,0 +1,30 @@
+parameters:
```

Comment:
or you could use digipolisgent/qa-drupal

Reply:
not as long that the rules are not ok